src/main/webapp/index.html

```diff
@@ -1,32 +1,36 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     <head>
         <meta charset="UTF-8">
         <title>PromoTube</title>
         <link rel="stylesheet" href="style.css">
-            <script src="script.js"></script>
+        <script src="script.js"></script>
+        <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet'>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```

Comment:
what's this integrity thing?... is it necessary?

Reply:
Not really sure... just copied from the website [instructions](https://getbootstrap.com/docs/4.3/getting-started/download/).